workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -45,6 +48,7 @@ public MutableView(World world, ILogDispatcher logDispatcher)
                 typeof(Action<Entity, ComponentType, object>), entityManager, setComponentObjectMethodInfo);
 
             FindTranslationUnits();
+            FindGameObjectTranslations();
```

Comment:
Is there any reason this call or the function needs to be in the mutable view? It looks like this would be better suited to being in the `GameObjectDispatcherCallbacksSystem`.

Reply:
Moved this into the `GameObjectDispatcherCallbacksSystem`.